cedar-policy/tests/public_interface.rs

```diff
@@ -219,11 +219,11 @@ fn authorize_custom_request() -> Result<(), Box<dyn Error>> {
         Decision::Allow
     );
 
-    // Requesting with an unspecified principal or resource will return Deny (but not fail)
+    // Requesting with default principal or resource will return Deny (but not fail)
```

Comment:
This test also makes it seem like `__cedar::"Default"` is something special. Are these tests really testing something interesting? I suspect they could be removed in favor of existing tests.

Reply:
Eh, I think it's nice to have a confirmation our migration suggestion is breaking anything